<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayInfoWidget</class>
 <widget class="QWidget" name="PlayInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>78</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>254</width>
    <height>78</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>254</width>
    <height>78</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="iFirstButton">
     <property name="minimumSize">
      <size>
       <width>95</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>95</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>First</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="iFrameNumberLabel">
     <property name="minimumSize">
      <size>
       <width>55</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>55</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="iLastButton">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Last</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="iPreviousButton">
     <property name="minimumSize">
      <size>
       <width>95</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>95</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="iPlayButton">
     <property name="minimumSize">
      <size>
       <width>67</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>67</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="iNextButton">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
